### DISCUSSION TITLE
LEDVANCE smart+ panel 120x30 not pairing

### DISCUSSION THREAD

**Discussion by arbakker** [Koenkk/zigbee2mqtt#6457] [Devices]
I am trying to pair a [LEDVANCE Smart+ Panel 120x30](https://www.amazon.nl/LEDVANCE-Armatuur-instelbaar-materiaal-aluminum/dp/B07VD9G7W7/) in my setup (zigbee2mqtt v1.17.0 and a Conbee2 stick with latest firmware on a raspberry pi) without any success. I am monitoring the zigbee2mqtt  logs while trying to [reset](https://www.ledvance.com/consumer/smart/faq/faq-for-smart-zigbee-products/index.jsp) the led panel (turning it on and off for 5 seconds for 5 consecutive times), but I do not see any events relating to a new device coming by (with `permit_join: true` in the zigbee2mqtt config, and also join enabled from the web interface). 

This device is not explicitly listed as supported, but the smaller version [is](https://www.zigbee2mqtt.io/devices/4058075181472.html). 

I think it is suspicious no log messages are showing up relating to a new device. If the device is not supported would an error message show? Or would it just fail to pair silently like in this particular case?  Also when the panel is turned on it keeps cycling between a dimmer and brighter light (every 10 seconds or so), not sure if this is an indication of the panel not being paired, or that panel is broken (in terms of zigbee 

> connectivity). 

Any thoughts on this issue are greatly appreciated, thanks!

**Comment by arbakker** [comment]
So I figured out my led panel is not in [`devices.js`](https://github.com/Koenkk/zigbee-herdsman-converters/blob/8f3326883819b8c8a34837275467d4612a43cce2/devices.js#L5056) and therefore probably ignored. The device id of my panel is `4058075181519`. So I suppose adding my device to `devices.js` will fix the issue ans immediately  add the other size variants as well, while we are at it.

**Comment by arbakker** [comment]
So I figured it would be wise to first read the documentation (well written btw), which is helpful in understanding what is going on. Concluding from the docs my previous assertion is incorrect:

>So I figured out my led panel is not in devices.js and therefore probably ignored. 

The device should pair, but unfortunately it does not. So now I am stuck at [step 1](https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html#1-pairing-the-device-with-zigbee2mqtt). I do not see anything showing up in the zigbee2mqtt logs after resetting the panel. I set `loglevel` to `debug` and `permit_join` to `true` in the zigbee2mqtt configuration.yaml. 

Any ideas on how to debug the panel are appreciated. I suspect the zigbee connectivity might be broken in the panel (the led panel itself works).  Any way of finding out if the zigbee module of the device is broken?

**Reply by mabosch74** [reply]
sure it's 5 times for 5 seconds on/off? try 5 seconds off, 2 seconds on, 5 times.

**Reply by arbakker** [reply]
Thanks for your suggestion, I tried it but unfortunately without results. I am quite sure the reset procedure should 5x 5s on, 5s off. See the [Smart+ FAQ](https://www.ledvance.com/consumer/smart/faq/faq-for-smart-zigbee-products/index.jsp) (section 2.3.5) :

> c) Resetting SMART+ Zigbee 3.0 Tunable White lamps:
To reset a SMART+ Zigbee 3.0 Tunable White lamp to factory setting, switch it ON for five seconds then OFF for five seconds five times in succession. When switched on again, the lamp flashes two times cold white and warm white as confirmation and can now be added to a smart home system again.

**Reply by mabosch74** [reply]
that's at all not correct.

Lightify/Smart+ has different settings for resetting:
Some modes:
5 times: 2 sec = on, 5 sec = off
5 times: 3 sec = on, 5 sec = off
5 times: 5 sec = on, 5 sec = off
this can be different by the installed firmware version. and the timing must be exactly, you may need to try it more than one time.

**Reply by arbakker** [reply]
Thanks for the suggestion, much appreciated. 

I contacted LEDVANCE support to see if they could shine some light on this (pun intended), and they confirmed the reset procedure should be:
- 5 times: 5 sec = on, 5 sec = off

I have tried the reset recommended procedure numerous times, with a stopwatch to get the timing right, but without any success. Maybe my timing is consistently off or the panel might be defective. 

Considering that did not work I also tried your suggestions, but also without any success.
